packages/ui/src/composites/wui-card-select/styles.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from 'lit'
+
+export default css`
+  button {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
```

Comment:
Would this component be responsible or does it actually have fixed width? We need to check with design team, either this button expands or stays constant and only spacing / gaps etc around it do